// tech-tip-cyber/index.js
'use strict';

var toString = require('to-str');
var randomLorem = require('random-lorem');
var randomDoamins = require('random-domains');

module.exports = function (options) {
  var domain = options && options.domain
    ? toString(options.domain)
    : randomDoamins();

  return randomLorem() + '@' + domain;
};

// Random
module.exports.ipAddress = require('./Functions/Random/ip')
module.exports.randomName = require('./Functions/Random/name')
module.exports.randomNumber = require('./Functions/Random/number')
module.exports.randomPassword = require('./Functions/Random/password')

// Other
module.exports.Capitalize = require('./Functions/Other/capital')
module.exports.chatBot = require('./Functions/Other/chatbot')
module.exports.fetchMessage = require('./Functions/Other/Transcript/transcript');
module.exports.timeout = require('./Functions/Other/timeout')
module.exports.YTDate = require('./Functions/Other/ytdate')
module.exports.YTDescription = require('./Functions/Other/ytdesc')
module.exports.YTSub = require('./Functions/Other/ytsub')
module.exports.YTVideo = require('./Functions/Other/ytvid')
module.exports.YTView = require('./Functions/Other/ytview')
module.exports.YTThumbNail = require('./Functions/Other/ytthumbnail')
